spec/fabricators/application_fabricator.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+Fabricator(:application, from: Doorkeeper::Application) do
+  name 'iOS client'
```

Comment:
We also can provide more variant of data by this way.

```suggestion
  name ['iOS client', 'Android client'].sample
```